### DISCUSSION TITLE
Workspaces and Layouts

### DISCUSSION THREAD

**Discussion by 2joy** [WerWolv/ImHex#1576] [Get Help]
Hi, I have a couple of questions about these entities:
1. What is the difference between them? In what situation should I use one or the other?
2. How do I rename or delete a workspace or layout I've created?

**Comment by WerWolv** [comment]
Hey
Layouts are basically presets or templates for how to setup a workspace. You can export them at any time and import them again later on.
Workspaces on the other hand store the current layout of your interface whenever you make a change. So you can quickly switch back and fourth between multiple workspaces and you keep the layout you had last when using that workspace

**Reply by 2joy** [reply]
Thank you. Regarding the second question. I figured out how to delete the layout. When you press the Shift key, a cross appears, clicking on which does the job. But this method does not work for workspaces. How can I remove them?

<img width="358" alt="image" src="https://github.com/WerWolv/ImHex/assets/47565434/45792933-4e3b-4b71-842c-bb9b220e0990">

> You can export them at any time and import them again later on.

How can I do this, I don’t see any suitable options in the export menu?

<img width="571" alt="image" src="https://github.com/WerWolv/ImHex/assets/47565434/9a4848e1-a694-4e4a-8c15-572ec42552f9">

**Reply by WerWolv** [reply]
With the latest commit, removing them the same way as layouts is possible now. Up until now it wasn't
